multibuildingdetector/__main__.py

```diff
@@ -36,12 +36,13 @@ def run(input, output, batch_size, iterator='SerialIterator', device=-1,
         chainer.cuda.get_device_from_id(device).use()
         model.to_gpu()
 
-    train = TransformDataset(
-        xmldataset.XMLDataset(input),
+    train, test = reader.load_train_test_set(input_dir, 'xml', 0.8)
```

Comment:
We should not look at the xml files only, but at the JPG files. If there are no bboxes on a picture, no xml exists, but we need them for the 'none' category (unless proven otherwise).

Reply:
Ok good point, but what if our files are mixed xml and json?

Reply:
I think we should, let the user configure the specific reader class and we just provide the interface with the `load_train_test` method. This way we do not have to care about the file endings etc.

Reply:
This can be done in a separate PR